plugins/inputs/couchbase/README.md

```diff
@@ -20,6 +20,14 @@ This plugin gets metrics for each Couchbase node, as well as detailed metrics fo
 
   ## Filter bucket fields to include only here.
   # bucket_stats_included = ["quota_percent_used", "ops_per_sec", "disk_fetches", "item_count", "disk_used", "data_used", "mem_used"]
+  
+  ## Whether to collect cluster-wide bucket statistics
+  ## It is recommended to disable this in favor of node_stats
+  ## to get a better view of the cluster.
+  cluster_bucket_stats = true
+  
+  ## Whether to collect bucket stats for each individual node
+  node_bucket_stats = false
```

Comment:
Instead of having 2 booleans can we set it as an array of strings string? This would make the plugin more extendable.

Reply:
Normally I would agree that an array of strings would be better. However, I don't think it's worth doing in this case since you can only collect bucket-level stats from the node or cluster. There are no other places to collect stats for buckets.